### PR TITLE
Repo maintenance / harmonization

### PR DESCRIPTION
**Pull request by vanduynslagerp**
- Switch from `standard` to `eslint` + `eslint-config-standard` + [prettier](https://github.com/prettier/prettier)
- Use ava and nyc
- Add a License file + badge
- Remove npm and yarn lockfile
- Use a more generic `.gitignore` that would work on Windows, Linux, Mac OS
- Set minimum node version to 4
- Add badges

@gr2m can you review and add this repo to codecov ? Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
-# semantic-release-error
+# **@semantic-release/error**
```
Comment (gr2m) [inline README.md]: I don’t think we need the `**` bold formatting when it’s already a level 1 heading

File: README.md [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+
+[![Travis](https://img.shields.io/travis/semantic-release/error.svg)](https://travis-ci.org/semantic-release/error)
+[![Codecov](https://img.shields.io/codecov/c/github/semantic-release/error.svg)](https://codecov.io/gh/semantic-release/error)
+
+## Usage
+
+```js
+cont SemanticReleaseError = require('@semantic-release/error');
```
Comment (gr2m) [inline README.md]: const

File: README.md (hunk 1 above)
Comment (vanduynslagerp) [reply README.md]: Removed

File: README.md (hunk 2 above)
Comment (vanduynslagerp) [reply README.md]: Good catch! fixed!

**Comment by gr2m** [comment]
Do you want to release a new version? The `fix(REAMDE): ...` commit will trigger one. Usually I’d only use `docs: ` for readme updates. I don’t mind releasing a new version, I’ll leave it up to you :)

**Comment by vanduynslagerp** [comment]
It's just to publish the changes of the readme to NPM. It's really minor but it keeps npm and the repo in sync.
USually I wouldn't do a release for such a small change, but considering the simplicity of this package we won't do another release anytime soon...

**Comment by gr2m** [comment]
for the record: we released it as `@semantic-release/error@2.0.0`:
https://github.com/semantic-release/error/releases/tag/v2.0.0

**Pull request merged**